### PR TITLE
#328 Fix 65535 string length issue

### PR DESCRIPTION
**Pull request by mkarebski**

### CODE REVIEW

File: pkg/controller/jenkins/configuration/user/casc/caac.go [hunk 1]
```diff
@@ -35,16 +35,49 @@ func (c *ConfigurationAsCode) Ensure(jenkins *v1alpha2.Jenkins) (requeue bool, e
...
+}
+
+def stream = new ByteArrayInputStream(configSb.toString().getBytes('UTF-8'))
 
 def source = new io.jenkins.plugins.casc.yaml.YamlSource(stream, io.jenkins.plugins.casc.yaml.YamlSource.READ_FROM_INPUTSTREAM)
 io.jenkins.plugins.casc.ConfigurationAsCode.get().configureWith(source)
 `
+
+func prepareScript(script string) string {
+	groovyUtf8MaxStringLength := 65535
```
Comment (tomaszsek) [inline pkg/controller/jenkins/configuration/user/casc/caac.go:56]: extract constant

File: pkg/controller/jenkins/configuration/user/casc/caac.go [hunk 2]
```diff
@@ -35,16 +35,49 @@ func (c *ConfigurationAsCode) Ensure(jenkins *v1alpha2.Jenkins) (requeue bool, e
...
+		slicedScript = splitTooLongScript(script)
+	} else {
+		slicedScript = append(slicedScript, script)
+	}
+
+	return strings.Join(slicedScript, "''','''")
+}
+
+func splitTooLongScript(groovyScript string) []string {
+	groovyUtf8MaxStringLength := 65535
```
Comment (tomaszsek) [inline pkg/controller/jenkins/configuration/user/casc/caac.go:68]: extract constant

File: pkg/controller/jenkins/configuration/user/casc/caas_test.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package casc
+
+import (
+	"github.com/stretchr/testify/assert"
+	"strings"
+	"testing"
```
Comment (tomaszsek) [inline pkg/controller/jenkins/configuration/user/casc/caas_test.go:6]:
```suggestion
	"strings"
	"testing"
	
	"github.com/stretchr/testify/assert"
```

File: pkg/controller/jenkins/configuration/user/casc/caac.go (hunk 2 above)
Comment (mkarebski) [reply pkg/controller/jenkins/configuration/user/casc/caac.go]: done.

File: pkg/controller/jenkins/configuration/user/casc/caas_test.go (hunk 3 above)
Comment (mkarebski) [reply pkg/controller/jenkins/configuration/user/casc/caas_test.go]: done.

File: pkg/controller/jenkins/configuration/user/casc/caac.go (hunk 1 above)
Comment (mkarebski) [reply pkg/controller/jenkins/configuration/user/casc/caac.go]: done

**Pull request merged**